client/src/app/pages/applications/components/application-form/application-form.tsx

```diff
@@ -274,8 +274,26 @@ export const ApplicationForm: React.FC<ApplicationFormProps> = ({
     }
   };
 
-  const onCreateUpdateApplicationSuccess = (response: any) => {
-    onSaved(response);
+  const onCreateUpdateApplicationSuccess = (
+    response: AxiosResponse<Application>
```

Comment:
I think we could use our custom useUpdateApplicationMutation & useCreateApplicationMutation hooks to 1) narrow this type to something like "AppName" & 2) Pass the name from the mutation function variable object param to the success handler.

Reply:
@ibolton336, generalizing this approach is, I believe out of scope of this PR, please let's discuss how we are going to do that here : https://github.com/konveyor/tackle2-ui/issues/1214.

Reply:
Np - we can address the missing entity name in the update notification later. However, I think it would make more sense to have two separate handlers for create / update here since there.

Reply:
@ibolton336, I agree it makes sense to have separate handlers. Let me change that.